lib/open_project/translations/helpers/tmp_directory.rb

```diff
@@ -0,0 +1,14 @@
+module TmpDirectory
+  def create_tmp_directory(delete: nil)
+    tmp_path = File.join(FileUtils.pwd, 'tmp')
+    if delete
+      FileUtils.rm_rf tmp_path
+    end
+    FileUtils.mkdir_p tmp_path
+    tmp_path
```

Comment:
I think you have the big block regardless of what you are doing.

I am fine with having an optional `debug` option if that helps you.